api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -62,7 +62,7 @@
 
 @RunWith(BeforeAfterSpringTestRunner.class)
 @Import({
-  BigQueryService.class,
+  BigQueryServiceTest.class,
```

Comment:
Just to clarify my understanding with what's going on here... this @Import list is what causes these classes to be autowired into the test case, so importing a subclass will cause BigQueryServiceTest to be injected anywhere that a BigQueryService is required in the execution test?

Reply:
This is correct. We are explicitly loading only the classes we are interested in.